### ISSUE TITLE
automate rebuilding paper.pdf during review process

### ISSUE THREAD

**Issue by danielskatz** [openjournals/joss#105]
As we make changes in paper.md and the bib entries, how does the compiled paper get updated? Or does it not until the review process ends? It might be nice to have some way that the original submitter could recompile, or a hook that recompiles on changes to specific files in the repo once a submission to JOSS has been made. If this makes sense, perhaps it should be in issue in the JOSS repo, rather than a comment here?

**Comment by arfon** [comment]
👍 definitely. Ideally I think this should be something that happens right at the start of the review (i.e. Whedon should go looking for the paper and compile it) and it should be something that anyone can request at any point during the review.

**Comment by arfon** [comment]
Fixed in https://github.com/openjournals/whedon-api/commit/a15bf22b3f27b0150d05792076c545fd39a4da76
